2018/csaw_quals/take_an_l/README.md

```diff
@@ -0,0 +1,35 @@
+# Take an L
+
+## Challenge details
+| Category | Points |
+|:---------|-------:|
+| misc     | 200    |
+
+### Description
+> Fill the grid with L's but avoid the marked spot for the W
+> The origin is at (0,0) on the top left
+
+## Write-up
+
+`description.pdf` holds further details of the problem.
+One important detail is that the grid is always `64 x 64`
+and we have 5 seconds to send a response.
+
+The solution is simple enough to express recursively:
+- base case: a `2 x 2` square with a mark is tiled in the obvious way
+- inductive step: if a `2^k x 2^k` square with a mark can be tiled,
+  we tile a `2^(k + 1) x 2^(k + 1)` square by splitting it into four
+  `2^k x 2^k` quadrants and tiling an L into quadrants which do not contain
+  a mark already so that each contains exactly one cell comprising the L.
```

Comment:
The wording here is a bit confusing --- just saying "tiling an L with one cell in each quadrant" as in the comments would be clearer